tests/unit/test_events.py

```diff
@@ -177,70 +177,6 @@ def distgit_commit(self):
         with open(DATA_DIR / "fedmsg" / "distgit_commit.json") as outfile:
             return json.load(outfile)
 
-    @pytest.fixture()
-    def koji_build_scratch_start(self):
-        with open(
-            DATA_DIR / "fedmsg" / "koji_build_scratch_start.json", "r"
-        ) as outfile:
-            return json.load(outfile)
-
-    @pytest.fixture()
-    def koji_build_scratch_end(self):
-        with open(DATA_DIR / "fedmsg" / "koji_build_scratch_end.json", "r") as outfile:
-            return json.load(outfile)
-
-    @pytest.fixture()
-    def koji_build_start_old_format(self):
-        with open(
-            DATA_DIR / "fedmsg" / "koji_build_start_old_format.json", "r"
-        ) as outfile:
-            return json.load(outfile)
-
-    @pytest.fixture()
-    def koji_build_start_rawhide(self):
-        with open(
-            DATA_DIR / "fedmsg" / "koji_build_start_rawhide.json", "r"
-        ) as outfile:
-            return json.load(outfile)
-
-    @pytest.fixture()
-    def koji_build_start_f35(self):
-        with open(DATA_DIR / "fedmsg" / "koji_build_start_f35.json", "r") as outfile:
-            return json.load(outfile)
-
-    @pytest.fixture()
-    def koji_build_start_epel8(self):
-        with open(DATA_DIR / "fedmsg" / "koji_build_start_epel8.json", "r") as outfile:
-            return json.load(outfile)
-
-    @pytest.fixture()
-    def koji_build_completed_old_format(self):
-        with open(
-            DATA_DIR / "fedmsg" / "koji_build_completed_old_format.json", "r"
-        ) as outfile:
-            return json.load(outfile)
-
-    @pytest.fixture()
-    def koji_build_completed_rawhide(self):
-        with open(
-            DATA_DIR / "fedmsg" / "koji_build_completed_rawhide.json", "r"
-        ) as outfile:
-            return json.load(outfile)
-
-    @pytest.fixture()
-    def koji_build_completed_f35(self):
-        with open(
-            DATA_DIR / "fedmsg" / "koji_build_completed_f35.json", "r"
-        ) as outfile:
-            return json.load(outfile)
-
-    @pytest.fixture()
-    def koji_build_completed_epel8(self):
-        with open(
-            DATA_DIR / "fedmsg" / "koji_build_completed_epel8.json", "r"
-        ) as outfile:
-            return json.load(outfile)
-
```

Comment:
would it make sense to drop unused files from the repo?

Reply:
The files are still in use. We just had the fixtures defined multiple times..;)